runtime/stdlib/account.go

```diff
@@ -1369,250 +1376,303 @@ func newAccountContractsChangeFunction(
 		gauge,
 		functionType,
 		func(invocation interpreter.Invocation) interpreter.Value {
+			return changeAccountContracts(invocation, handler, addressValue, isUpdate)
```

Comment:
The code starting from here was extracted out to a function, so that it can be used in both `update` function as well as `tryUpdate` function.

It was moved as-is, and no change was made to the logic.

-- Refactor starts here  --